test/NuGet.Clients.Tests/NuGet.CommandLine.Test/FeedPackagePruningTests.cs

```diff
@@ -66,7 +66,7 @@ public async Task FeedPackagePruning_GivenThatAV3FeedPrunesAPackageDuringRestore
                 server.Start();
 
                 var feedUrl = server.Uri + "index.json";
-
+                pathContext.Settings.AddSource(feedUrl, feedUrl, allowInsecureConnectionsValue: "true");
```

Comment:
Is there a tracking issue to change all the test dependencies to use HTTPS?

Reply:
No. I think that is a good point.